<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QSplitter" name="centralwidget">
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="qfcmd::FsTreeView" name="treeView"/>
   <widget class="QSplitter" name="pannel">
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="qfcmd::FsTabWidget" name="leftPanel">
     <property name="movable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="qfcmd::FsTabWidget" name="rightPanel">
     <property name="movable">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionShowToolbar"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionPerferences"/>
   </widget>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionShowToolbar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Toolbar</string>
   </property>
  </action>
  <action name="actionPerferences">
   <property name="text">
    <string>Perferences</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qfcmd::FsTabWidget</class>
   <extends>QTabWidget</extends>
   <header>widget/fstabwidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qfcmd::FsTreeView</class>
   <extends>QTreeView</extends>
   <header>widget/fstreeview.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
